module1.js

```diff
@@ -152,11 +170,16 @@ function question1(name1) {
   @returns { totalPrice } a string containing the currency sign too
 */
 
-// function question9( retailPrice ) {
-//   // TODO
-  
-//   return /* ??? */;
-// }
+function question9( retailPrice, taxAmount ) {
+  var retailPrice1 = retailPrice.replace(/[^0-9\.]+/g,"");
+  var retailPrice2 = parseInt(retailPrice1)
+  var tax1 = (20 * retailPrice1)/100
+  var final = retailPrice2 + tax1
+  // TODO
+
+  return final;
+    }
+
```

Comment:
A little longer than some I've seen, but the use of the regex is a good option - where did you find that?